.github/cspell.json

```diff
@@ -0,0 +1,8 @@
+{
+    "version": "0.1",
+    "language": "en",
+    "flagWords": [],
+    "ignoreWords":["broadphase","subdir","Hitbox","Raycast","positionable","Renderable","pallete","raytracing"] ,
```

Comment:
1. `pallete` is a typo - should be "palette". Also, `subdir` is just an abbreviation of "subdirectory".
2. This looks like it'll end up being a rather long list, so let's make it one-word-per-line to make future edits easier.
3. Is there a difference between capitalized and lowercase words? If not, then probably all words should be turned into lowercase.

Reply:
1-> just added few words from the output of the spell checker
2-> sure

3-> it is case insensitive (https://github.com/streetsidesoftware/cspell/tree/main/packages/cspell#things-to-note)

Reply:
What is left to do here?

Reply:
There are 2526 issues found by the analyzer :)

Reply:
What should we do to solve this?
Adding every word in ignore words  list?

Reply:
I guess so, but we'll have to make sure to not add actual typos to the ignore list so we can't just do a blanket add.

Reply:
We can use the spell checker to check our ignore words list then 😂

Reply:
I can see plenty of typos reported: `programatically`, `operatiosn`, `paramater`, `slided`, `everytime`, `Uncontolled`, etc. -- that's just among the last 20.

Reply:
I will try to add as much i can in the list and also correct them in the files.